LoginPage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Formik, Field, Form } from 'formik';
+import { useHistory } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const LoginPage = () => {
+  const history = useHistory();
+
+  return (
+    <div className="container-fluid">
+      <div className="row">
+        <div className="col-md-6">
+          <img src="image.jpg" alt="Learning" style={{ height: '100vh', width: '100%' }} />
+        </div>
+        <div className="col-md-6">
```

Comment:
Consider moving the Formik form to a separate component for better modularity and readability.